packages/core/core/src/migrations/database/5.0.0-discard-drafts.ts

```diff
@@ -77,15 +77,15 @@ async function copyPublishedEntriesToDraft({
    */
   await trx
     // INSERT INTO tableName (columnName1, columnName2, columnName3, ...)
-    .into(trx.raw(`${meta.tableName} (${scalarAttributes.join(', ')})`))
+    .into(trx.raw(`?? (??)`, [meta.tableName, scalarAttributes]))
```

Comment:
I'm not sure you can spread an array into a single ?? identifier 🤔 you might need to do sth like 

```
trx.raw(`?? (${scalarAttributes.map(() => `??`).join(', ')})`, [meta.tableName, scalarAttributes])
```

Reply:
Ahhh, you're right. I wonder why it actually works though? Undocumented feature? 😆

I'll fix it just to be safe.